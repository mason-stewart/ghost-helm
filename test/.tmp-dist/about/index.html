<!DOCTYPE html>
<html>
  <head>
    <title>Iron Yard- About</title>
    <meta name="description" content="The Iron Yard exists to create exceptional value for people and their ideas through code education, startup accelerators, and coworking spaces."/>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- Place favicon.ico and apple-touch-icon.png in the root directory-->
    <link rel="stylesheet" href="/styles/vendor.css">
    <link rel="stylesheet" href="/styles/main.css">
    <script src="/scripts/vendor/modernizr.js"></script>
    <script type="text/javascript" src="//use.typekit.net/zza7guc.js"></script>
    <script type="text/javascript">try{Typekit.load();}catch(e){}</script>
  </head>
  <body>
    <div class="container-fluid gray-dark-bg">
      <div class="header-dimmer gray-dark-bg">
        <div class="container">
          <div class="header">
            <div class="logo"><a href="http://theironyard.com"><img src="/images/iron-yard-logo.svg" style="preserveAspectRatio=MinYMin slice"/></a></div>
            <div class="nav">
              <div class="primary-nav">
                <ul>
                  <li><a href="/locations">Locations</a></li>
                  <li><a href="/academy">Academy</a></li>
                  <li><a href="/accelerator">Accelerator</a></li>
                  <li><a href="/events">Events</a></li>
                </ul>
              </div>
              <div class="secondary-nav">
                <ul>
                  <li><a href="/about">About</a></li>
                  <li><a href="/jobs">Jobs</a></li>
                  <li><a href="http://news.theironyard.com">News</a></li>
                  <li><a href="http://blog.theironyard.com">Blog</a></li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div style="" class=" ">
        <div class="container">
          <div class="sub-nav">
            <div class="col-xs-12">
              <h2>About The Iron Yard</h2>
              <ul>
                <li><a href="/about/team">Team</a></li>
                <li><a href="/about/culture">Culture</a></li>
              </ul>
              <div class="triangle gray-dark-triangle"></div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-md-8">
          <h3>The Iron Yard exists to create exceptional value for people and their ideas through code education, startup accelerators and coworking spaces.</h3>
          <p>Through our broad, deep and growing network of teachers, programmers, designers, entrepreneurs, investors, employers and coworking spaces, we have developed a thriving community where serendipity happens en masse.</p>
          <h4>We are people-centric.</h4>
          <p>We put our students, teams, investors and supporters first in all of our decision making. Our success is entirely dependent on their success.</p>
          <h4>Our challenge is immense.</h4>
          <p>The world has huge problems that need to be solved, many of which technology is perfectly suited to address. Companies around the globe are answering this call, but anyone with boots on the ground can feel the acute shortage of talent needed to rise to the occasion. That gap is widening right now, and over the next decade the shortfall will affect industries, economies, businesses and individuals as they fight to add value in a digital world.</p>
          <h4>The Iron Yard is a solution at that juncture of supply, demand, business and a broken education system.</h4>
          <p>We are committed to training the highest-quality developers and startups in the world, both in skill and in character. Our mission is to find people who want to pursue the craft and life-long adventure of technology, teach them the tools of the trade, and then release them into the world with the drive and capability to make a difference. We believe that passionate, talented people who love what they do will create great families, jobs, companies and solutions to the world’s problems.</p>
          <h4>Join us.</h4>
          <p>We want to accomplish our goal alongside talented, passionate people who share our vision, love what they do, and want to have a whole lot of fun along the way. <a href="/jobs/">Check out our job openings</a> and join the team.</p>
        </div>
        <div class="col-md-4">
          <h2>Connect</h2>
          <ul>
            <li><a href="/jobs/">Join our team</a></li>
            <li><a href="mailto:hello@theironyard.com" target="_blank">hello@theironyard.com </a></li>
            <li><a href="http://twitter.com/theironyard" target="_blank">@theironyard </a></li>
            <li><a href="/locations">Locations </a></li>
            <li><a href="/about/team">Meet our team</a></li>
            <li><a href="/about/culture">Discover our culture</a></li>
          </ul>
          <hr><img src="/images/about/about-page-team.jpg" style="max-width:500px" class="photo"><img src="/images/about/about-page-collaborate.jpg" style="max-width:500px" class="photo"><img src="/images/about/about-page-mason.jpg" style="max-width:500px" class="photo">
        </div>
      </div>
    </div>
    <div class="container-fluid gray-dark-bg">
      <div class="footer">
        <div class="row">
          <div class="logo"><img src="/images/footer-logo.png"></div>
          <div class="footer-nav">
            <ul>
              <li><a href="/locations">Locations</a></li>
              <li><a href="/academy">Academy</a></li>
              <li><a href="/accelerator">Accelerator</a></li>
              <li><a href="/events">Events</a></li>
              <li><a href="/about">About</a></li>
              <li><a href="/jobs">Jobs</a></li>
              <li><a href="http://news.theironyard.com">News</a></li>
              <li><a href="http://blog.theironyard.com">Blog</a></li>
            </ul>
          </div>
          <div class="social-nav">
            <ul>
              <li><a href="http://twitter.com/theironyard" class="ss-icon">twitter</a></li>
              <li><a href="http://facebook.com/theironyard" class="ss-icon">facebook</a></li>
              <li><a href="http://vimeo.com/theironyard" class="ss-icon">vimeo	</a></li>
            </ul>
          </div>
          <div class="copyright">©2014 The Iron Yard. All Rights Reserved   </div>
        </div>
      </div>
    </div>
    <!-- Google Analytics: change UA-XXXXX-X to be your site's ID.-->
    <script type="text/javascript">
      (function(b,o,i,l,e,r){b.GoogleAnalyticsObject=l;b[l]||(b[l]=
      function(){(b[l].q=b[l].q||[]).push(arguments)});b[l].l=+new Date;
      e=o.createElement(i);r=o.getElementsByTagName(i)[0];
      e.src='//www.google-analytics.com/analytics.js';
      r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
      ga('create','UA-32355916-2');ga('send','pageview');
      
    </script>
    <script src="/scripts/vendor.js"></script>
    <script src="/scripts/main.js"></script>
    <script>var reg = /city=([^&]*)\//g;
var city = reg.exec(window.location.search);
if (city) {
  $('.class-table tr').removeClass('active');
  $('.class-table tr.' + city[1]).first().prependTo('.class-table tbody').addClass('active');
}

    </script>
    <!--browser update warning-->
    <script type="text/javascript">
      var $buoop = {
        vs:{i:9,f:15,o:12.1,s:5.1},
        text: "Your browser is out of date. Please <a href=\"https://browser-update.org/update.html\">Update Your Browser</a> for more security, comfort and the best experience on this site.",
        newwindow: true
      }; 
      $buoop.ol = window.onload; 
      window.onload=function(){ 
        try {if ($buoop.ol) $buoop.ol();}catch (e) {} 
        var e = document.createElement("script"); 
        e.setAttribute("type", "text/javascript"); 
        e.setAttribute("src", "//browser-update.org/update.js"); 
        document.body.appendChild(e); 
      }
      
    </script>
    <!--clicky analytics-->
    <script src="//static.getclicky.com/js"></script>
    <script type="text/javascript">try{ clicky.init(100714900); }catch(e){}</script>
  </body>
</html>