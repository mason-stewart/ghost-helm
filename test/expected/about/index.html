<!DOCTYPE html>
<html>
  <head>
    <title>Great</title>
    <meta name="description" content="Wonderful"/>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="/styles/vendor.css">
    <link rel="stylesheet" href="/styles/main.css">
    <script src="/scripts/vendor/modernizr.js"></script>
  </head>
  <body>
    <div class="container-fluid gray-dark-bg">
      <p>Wow!</p>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-md-8">
          <h3>Cool</h3>
          <p>Wow</p>
        </div>
      </div>
    </div>
    <div class="container-fluid gray-dark-bg">
      <div class="footer">
        <div class="row">
          <div class="copyright">Â©2014 Programming Is A Cool Job. All Rights Reserved.  </div>
        </div>
      </div>
    </div>
    <script type="text/javascript">
      function(){}
      
    </script>
    <script src="/scripts/vendor.js"></script>
    <script src="/scripts/main.js"></script>
  </body>
</html>